### DISCUSSION TITLE
Issue running print(fit)

### DISCUSSION THREAD

**Discussion by ben-kuzma-phd** [nlmixr2/nlmixr2#164] [Q&A]
When running the example in the landing page and ensuring all appropriate packages are installed, I get all the way to "print(fit)" and then it throws the following error: 

`Error in ifelse(use.utf(), "2"): argument "no" missing, with no default`

I've tried to track the issue down, and the model runs as I can plot it, but I just cannot see the output. I assume it might have to do with the 2 in nlmixr2. Anyone else have the same issue? 

I am running

RStudio 2022.12.0+353 "Elsbeth Geranium" Release
RStudio/2022.12.0+353 Chrome/102.0.5005.167 Electron/19.1.3 Safari/537.36

on windows

**Comment by mattfidler** [comment]
Yes this has been fixed in the CRAN versions of `nlmixr2` related libraries (`nlmixr2est` which many depend on a newer version of other packages too).

You can see a work-around here:

https://github.com/nlmixr2/nlmixr2est/issues/289#issuecomment-1318999433

You can install everything to the latest with:

```r
install.packages(c("dparser", "lotri", "rxode2ll", "rxode2parse",
                   "rxode2random", "rxode2et", "rxode2",
                   "nlmixr2data", "nlmixr2est", "nlmixr2extra",
                   "nlmixr2plot", "nlmixr2"))
```
